cmd/k8s-operator/operator.go

```diff
@@ -378,6 +379,7 @@ func runReconcilers(opts reconcilerOpts) {
 	nameserverFilter := handler.EnqueueRequestsFromMapFunc(managedResourceHandlerForType("nameserver"))
 	err = builder.ControllerManagedBy(mgr).
 		For(&tsapi.DNSConfig{}).
+		Named("dnsconfig-reconciler").
```

Comment:
I think keeping the existing name would have been more consistent - we do call this nameserver reconciler everywhere else. It does not really matter that the primary resource that we reconcile are DNSConfigs, I think the name does not have to be CRD-specific

Reply:
Fair enough, fixed in 121e461691160a0dd73e3deaaaa1332f88ea7cf4